apache-jena-osgi/jena-osgi/src/main/java/org/apache/jena/osgi/Activator.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.jena.osgi;
+
+import org.apache.jena.system.JenaSubsystemRegistry;
+import org.apache.jena.system.JenaSubsystemRegistryBasic;
+import org.apache.jena.system.JenaSystem;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+public class Activator implements BundleActivator {
+	
+	/* 
+	 * Based on http://svn.apache.org/repos/asf/aries/trunk/spi-fly/spi-fly-examples/spi-fly-example-provider-consumer-bundle/src/main/java/org/apache/aries/spifly/pc/bundle/Activator.java
+	 * the Activator#start() waits for bundle extension by Aries SPI Fly, configures JenaSystem logging and requests for initialization.     
+	 */
+	public void start(BundleContext context) throws Exception {
+
+		Thread t = new Thread(new Runnable() {
```

Comment:
The asynchronous initialisation was a tweak to wait for SPI Fly to enhance the bundle and wire the service providers.  This is obsolate now, since JenaSubsystemRegistryBasic passes the bundle's private classloader to ServiceLoader for their resolution. The current code in bundle Activator now simply calls the default initialisation (more an example).